README.md

```diff
@@ -0,0 +1,9 @@
+Sublime LSP plugin for Grammarly
+=
+
+[Grammarly](https://grammarly.com/) support for [Sublime's LSP plugin](https://github.com/sublimelsp/LSP), using the [Grammarly language server](https://github.com/znck/grammarly)
+
+Configuration
+==
+
+Make sure you have a recent `node` installed (17.0.0+ recommended) and accessible in sublime's path, even if sublime does not source your `~/.bashrc` (or equivalent).
```

Comment:
I would change the recommendation to 16.13.0+ as that is what is defined as minimum in the `plugin.py` itself. And also because in Node.js versioning, even versions are considered stable while odd are not.
And v16 is the current LTS.

Reply:
I will wait for [sublimelsp/lsp_utils#87](https://github.com/sublimelsp/lsp_utils/pull/87) to be merged and then I will remove it. (I added that note exactly because lsp_utils was not satisfying the dependency if the system one was insufficient)

Reply:
I tried with the updated lsp_utils. It seems to work like a charm, so I removed the paragraph. Thank you a lot for fixing it upstream!